lib/commands/general.js

```diff
@@ -16,8 +16,52 @@ commands.active = async function () {
 /**
  * Close app (simulate device home button). If a duration is given, app will
  * re-open after that many seconds
+ *
+ * https://github.com/appium/appium/issues/7741
+ *
+ * Possible values for `timeout`:
+ * - any positive number of seconds: come back after X seconds, show deprecation warning
+ * - any negative number of seconds: never come back, show deprecation warning
+ * - undefined: come back after the default timeout (defined by WDA), show deprecation warning. After deprecation: never come back
+ * - {timeout: 5000}: come back after 5 seconds
+ * - {timeout: null}, {timeout: -2}: never come back
  */
-commands.background = async function (seconds) {
+commands.background = async function (duration) {
+  let seconds;
+  if (_.isUndefined(duration)) {
+    // TODO: Replace the block after deprecated stuff is removed
+    // seconds = Number.MAX_SAFE_INTEGER;
+    log.warn(`commands.background: Application under test will never be restored in the future if no duration is provided.\n
```

Comment:
A template that spans two lines will keep the formatting. So this will have two newlines in it. Also, it will get printed weirdly in our logs, since the prefix will be skipped. I'd rather see this on a single line.